<launch>

<node name="bridge" pkg="ur3_control" type="bridge" respawn="false" output="screen">
  </node>
  <node name="arucoPanel3" pkg="simulation-cv" type="arucoPanel3.py" respawn="false" output="screen">
  </node>
  
  <!--
  <node name="panoramic3" pkg="simulation-cv" type="panoramic3.py" respawn="false" output="screen">
  </node>
  -->
  
  <node name="ERC" pkg="ur3_control" type="ERC" respawn="false" output="screen">
  </node>
  
  
  <node name="user_interface" pkg="ur3_control" type="user_interface.py" respawn="false" output="screen">
  </node>
  
</launch>
